docs/apps/architecture/query-builder.md

```diff
@@ -34,3 +34,39 @@ Common use cases include: Database Administration Tools, Web Application, Report
 8.Click on `Save` and then `Run`. It will display a query output.
 
 <img src= "/apps/img/query5.png">
+
+### G3 Query Builder
+
+If you wish to generate a graph, for example from past 10 days, you can use the feature `start_date`.
```

Comment:
When creating a query, you can select the data source as G3, which is helpful when you intend to visualize the data as a graph. The start_date feature facilitates specifying time ranges using natural language parsing in JavaScript.

Reply:
done